examples/transmissions/create_transmission_with_attachment.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Examples\Transmissions;
+
+require dirname(__FILE__).'/../bootstrap.php';
```

Comment:
As discussed, replacing this with an inline options object seems like it'd work better with my assumed "copy example and hack to fit" use case.